_pages/accessibility.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: sidenav
+redirect_from: https://design.login.gov/accessibility
```

Comment:
I don't expect this will do anything here in the context of this repository, since the configuration is for `jekyll-redirect-from` plugin, only affecting the build of this site, not [`identity-style-guide`](https://github.com/18F/identity-style-guide). 

Instead, should we leave this out here, then follow-up with a pull request in `identity-style-guide`, to be merged after the new brochure site is launched?

```suggestion
```